### DISCUSSION TITLE
Provide more meaningful changelogs

### DISCUSSION THREAD

**Discussion by s0enke** [superwerker/superwerker#188] [General] [answered]
Currently, the changelog is created from commit messages which tend to create non-meaningful changelogs by the automated release process.

Examples:

- https://github.com/superwerker/superwerker/releases/tag/v0.11.0 release for the #139 feature which contains all/most commit messages but not the feature itself 
- https://github.com/superwerker/superwerker/releases/tag/v0.10.0 which contains a message "reformat dashboard and add links to sso/cloudtrail/guardrails", but those links have been removed from the PR later on, so the message is misleading

Possible solutions:

1.  manually create the changelogs (this would probably depend on #158)
2. write commits messages with more discipline and/or squash-merge PRs so that the generated changelog contains less noise

What do you think?

**Comment by mavogel** [comment]
I think we'd try option 2 and see how it works with the outcome (if the changelog is more readable a satisfying). If no we can still go with option 1. WDYT?

**Comment by s0enke** [comment, marked as answer]
Let's  try option 2, so that we don't need another manual step when releasing. We also added the commit groups (e.g. features or fixes) to the auto-generated release notes: https://github.com/superwerker/superwerker/commit/d4ecf793d8d9aa8727bc8808bc4ba28a423cf3f2
